h2o-py/h2o/h2o.py

```diff
@@ -1031,12 +1031,13 @@ def download_csv(data, filename):
         f.write(urlopen()(url).read())
 
 
-def download_all_logs(dirname=".", filename=None):
+def download_all_logs(dirname=".", filename=None, container=None):
     """
     Download H2O log files to disk.
 
     :param dirname: a character string indicating the directory that the log file should be saved in.
-    :param filename: a string indicating the name that the CSV file should be. Note that the saved format is .zip, so the file name must include the .zip extension.
+    :param filename: a string indicating the name that the CSV file should be. Note that the default container format is .zip, so the file name must include the .zip extension.
+    :param container: a string indicating how to archive the logs, choice of "ZIP" and "TXT" (all logs concatenated together)
```

Comment:
I just wonder whether `.log` extension is not more common for logs than `.txt` ?

Reply:
hmm, good point - I think it does make sense to change it to LOG